### PR TITLE
Pre-release 3.2.2rc0 to rc branch 

### PR DESCRIPTION
**Pull request by bobleesj**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channel_targets:
+  - conda-forge diffpy.structure_rc
```
Comment (bobleesj) [inline recipe/conda_build_config.yaml:2]: Instructed in https://conda-forge.org/docs/maintainer/knowledge_base/#creating-a-pre-release-build

**Comment by bobleesj** [comment]
Ready for review @sbillinge Then, 

The following should be then available for us to test:

```
conda install -c conda-forge/label/diffpy.structure_rc -c conda-forge diffpy.structure
```

**Comment by sbillinge** [comment]
Ok, let's test.  We didn't request to rerender it, but I merged anyway.

**Comment by bobleesj** [comment]
Ah perhaps, we should have re-rendered this. 

```
conda install diffpy.structure
```

installs `3.2.2rc0`

From the doc:

> A rerender needs to happen for these changes to reflect in CI files. The channel_targets entries map

Learning...

since we will be recutting `diffpy.structure`, shall we do `3.2.2.rc1` release on PyPI then?

**Comment by bobleesj** [comment]
`diffpy.structure` conda install works. 

```
imac@imacs-iMac playground % python diffpy_structure_version.py 
diffpy.structure: 3.2.2rc0
```

**Comment by sbillinge** [comment]
> Ah perhaps, we should have re-rendered this.
> 
> ```
> conda install diffpy.structure
> ```
> 
> installs `3.2.2rc0`
> 
> From the doc:
> 
> > A rerender needs to happen for these changes to reflect in CI files. The channel_targets entries map
> 
> Learning...
> 
> since we will be recutting `diffpy.structure`, shall we do `3.2.2.rc1` release on PyPI then?

I am out so can do it tomorrow...

**Comment by sbillinge** [comment]
Let's do the new pypi tomorrow and repost to CF after a rerender so we are asking or friends to test what we actually think is the next release.  No point bothering them more than we have to.

**Comment by sbillinge** [comment]
It probably works tbh, but let's just do it right and then test.

**Comment by bobleesj** [comment]
okay. will continue tmr!

**Pull request merged**